runner/runners/invoker.go

```diff
@@ -0,0 +1,137 @@
+package runners
+
```

Comment:
Right now it's tested as part of the runner end-to-end tests. I'm happy to add tests if we think this is the right structure/decomposition.